### ISSUE TITLE
Feature: Add ARIA attributes to all forms

### ISSUE THREAD

**Issue by Avinash905** [SauravMukherjee44/Aec-Library-Website#1445]
### Feature description

ARIA attributes are crucial for enhancing web application accessibility, benefiting users with disabilities. Currently, the login, signup, contact us, issue forms lack these attributes, hindering proper interpretation by assistive technologies.

**Some steps to implement this enhancement-**

- Identify the form elements in your codebase.
- Determine the relevant ARIA attributes to add for each form element.
- Add ARIA roles to the forms and their elements (e.g., "form" role for the main form element).
- Include concise and descriptive ARIA labels using the aria-label attribute.
- Use the aria-describedby attribute to provide additional instructions for the forms.
- Implement ARIA states and properties (e.g., aria-required for required fields).

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/SauravMukherjee44/Aec-Library-Website/blob/main/CODE_OF_CONDUCT.md)

**Comment by Avinash905** [comment]
Please assign me this issue so that I can work on this issue under **GSSOC'23**.
